packages/training/deploy/docker/Dockerfile.base

```
@@ -0,0 +1,72 @@
+    peft>=0.8.0 \
+    accelerate>=1.12.0 \
+    atroposlib \
+    wandb \
+    huggingface-hub \
+    tenacity \
+    python-dotenv \
+    psycopg2-binary \
+    aiohttp
+
+# Verify installations
+RUN python3 -c "import torch; print(f'PyTorch {torch.__version__}, CUDA available: {torch.cuda.is_available()}')"
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)